src/test/kotlin/ua/marchenko/artauction/config/web/ExceptionHandlerTestConfiguration.kt

```diff
@@ -0,0 +1,30 @@
+package ua.marchenko.artauction.config.web
+
+import org.springframework.boot.test.context.TestConfiguration
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Profile
+import org.springframework.security.config.web.server.ServerHttpSecurity
+import org.springframework.security.web.server.SecurityWebFilterChain
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RestController
+import reactor.core.publisher.Mono
+
+@TestConfiguration
+class ExceptionHandlerTestConfiguration {
```

Comment:
Could be inlined into `ExceptionHandlerTest` class:

class ExceptionHandlerTest {


    class ExceptionHandlerTestConfiguration { ... }

    @RestController
    class ExceptionHandlerTestController { ... }
}